README.md

```diff
@@ -3,7 +3,49 @@
 # App Scaffold
 
 ## Description
-This repo contains a scaffold to help developers build [apps for Zendesk products](https://developer.zendesk.com/apps/docs/apps-v2/getting_started). The App Scaffold includes many features to help you maintain and scale your app. Some of the features provided by the App Scaffold are listed below, however you don't need prior experience in any of these to be able to use the scaffold successfully.
+This repo contains a scaffold to help developers build [apps for Zendesk products](https://developer.zendesk.com/apps/docs/apps-v2/getting_started).
+
+## Getting Started
+
+### Dependencies
+- [Node.js](https://nodejs.org/en/) >= 6.3.x
+- [Ruby](https://www.ruby-lang.org/) >= 2.0.x
+
+### Setup
+1. Clone or fork this repo
+2. Run `npm install`
+
+To run your app locally in Zendesk, you need the [Zendesk Apps Tools (ZAT)](https://github.com/zendesk/zendesk_apps_tools).
+
+You'll also need to run a couple of command-line Node.js-based tools that are installed using `npm`. For a node module to be available from the command-line, it must be installed globally.
+
+To setup these and other dependencies, run these commands:
+
+```
+gem install zendesk_apps_tools
+npm install --global webpack foreman karma-cli
+```
+
+Note: Foreman was originally created as a Ruby tool. If you prefer, you can install it by `gem install foreman` instead.
+
+## Running locally
+
+_Note: the App Scaffold currently depends on zat v1.35.12 or greater._
+
+Foreman allows you to easily run multiple processes in one tab. One process is `zat server --path=./dist`, which serves the app in a way that can be run in a supported Zendesk product. The second is `webpack --watch` to rebuild the project whenever you save changes to a source file.
+
+To run these processes, run
+
+```
+nf start
+```
+
+or run the individual commands from the Procfile in separate terminals.
+
+Note: if you installed the Ruby version of foreman, you'll need to use `foreman start`.
+
+## But why?
+The App Scaffold includes many features to help you maintain and scale your app. Some of the features provided by the App Scaffold are listed below, however you don't need prior experience in any of these to be able to use the scaffold successfully.
```

Comment:
Original:
>  Some of the features provided by the App Scaffold are listed below, however you don't need ...

Can't separate two independent clauses with a comma. It's called a comma splice error. Suggested edit:

>  Some of the features provided by the App Scaffold are listed below. However, you don't need ...